libs/langchain/langchain/document_loaders/gcs_file.py

```diff
@@ -62,6 +62,8 @@ def load(self) -> List[Document]:
         bucket = storage_client.get_bucket(self.bucket)
         # Create a blob object from the filepath
         blob = bucket.blob(self.blob)
+        # retrieve custom metadata associated with the blob
+        metadata = bucket.get_blob(self.blob).metadata
```

Comment:
can we just do `metadata = blob.metadata`?

Reply:
@baskaryan 
`blob()` does not actually send any request to GCS  while `get_blob()` fetches meta-information about the blob from GCS. So `metadata = blob.metadata` wont fetch any meta information. 

What I can think of instead of `blob = bucket.blob(self.blob)` we can have `blob = bucket.get_blob(self.blob)` and fetch metadata using `blob.metadata`. Your thoughts please

Reply:
@baskaryan any thoughts

Reply:
@sbusso @jarib @zeke need your thoughts on the above proposed solution

Reply:
gotcha, missed the blob vs get_blob part!